R/score_untrained.R

```diff
@@ -32,16 +33,18 @@ score_untrained <- function(single_task_data, oracle_output_data, model_id_list,
   # models in the single_task_data
   models <- unique(single_task_data$model_id)
   missing_model <- setdiff(model_id_list, single_task_data$model_id)
+
+  # `ens_fun` is a function from hubEnsembles specified by `ensemble_fun`
+  ens_fun <- getFromNamespace(ensemble_fun, ns = asNamespace("hubEnsembles"))
+
   # Compute importance score when importance_algorithm is 'lomo'
   if (importance_algorithm == "lomo") {
-    # `ens_fun` is a function from hubEnsembles specified by `ensemble_fun`
-    ens_fun <- getFromNamespace(ensemble_fun, ns = asNamespace("hubEnsembles"))
+    # build an ensemble with the specified method using all models available
     ens_all <- ens_fun(single_task_data,
       weights = NULL,
       model_id = "ensemble-all",
       ...
     )
-
     # build ensemble forecasts by leaving one model out
     ens_lomo <- lapply(models, function(x) {
```

Comment:
Is there a reason that we use paralellization for LOSOMO but not LOMO? It seems like they both have the potential to be fairly time-consuming, right?